### ISSUE TITLE
Workflow for contributions to the wiki/tutorial

### ISSUE THREAD

**Issue by greole** [ginkgo-project/ginkgo#582]
Some parts of the wiki/tutorial are currently stubs. I would like to help updating it, but don't have write access. What is your preferred way for contributions to the wiki?

**Comment by hartwiganzt** [comment]
@greole the easiest for now would be to create a private fork and create merge requests.

**Comment by pratikvn** [comment]
@greole , @fritzgoebel did a bit of updating of our Wiki. What he did was create a repo called ginkgo wiki https://github.com/fritzgoebel/ginkgo_wiki/wiki and updated the wiki there by copying from our wiki. We could all then review it and then he updated the ginkgo-project/ginkgo wiki

**Comment by greole** [comment]
@pratikvn Ok, I created a own copy of the wiki  at https://github.com/greole/ginkgo-wiki. There I can edit it and if you like the changes someone with write access can then push the changes back to https://github.com/ginkgo-project/ginkgo.wiki.git.

**Comment by pratikvn** [comment]
Yes, unfortunately, there does not seem to be a way to create a PR for the wiki. You can edit your repo and whenever you have a page that you want us to review, feel free to ping @ginkgo-project/reviewers and we can review and update the page into Ginkgo.
